File: pset6/mario.py
import cs50

def main():
    while True:
        print('Height: ', end='')
        height = int(input())
        if height <= 23 and height >= 0:
            break
    blocks = 2
    spaces = height - 1
    for i in range(height):
        for j in range(spaces):
            print(" ", end="")
        spaces = spaces - 1
        for k in range(blocks):
            print("#", end="")
        blocks = blocks + 1
        print("")

if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    while(true)
    {
        printf("Height: ");
        height = get_int();
        if (height <= 23 && height >= 0)
            break;
    }
    
    int blocks = 2;
    int spaces = height - 1;
    for(int i = 0; i < height; i++)
    {
        for(int j = 0; j < spaces; j++){
            printf(" ");
        }
        spaces = spaces - 1;
        for(int j = 0; j < blocks; j++){
            printf("#");
        }
        blocks = blocks + 1;
        printf("\n");
    }
}